src/gluonts/model/n_beats/_network.py

```diff
@@ -666,7 +694,8 @@ def __init__(self, *args, **kwargs) -> None:
 
     # noinspection PyMethodOverriding,PyPep8Naming
     def hybrid_forward(
-        self, F, past_target: Tensor, future_target: Tensor = None
+            self, F, past_target: Tensor, future_target: Tensor = None,
+            past_observed_values: Tensor = None, future_observed_values: Tensor = None,
```

Comment:
Maybe the optional arguments (that can be `None`) should be `Optional[Tensor]`, but I'm not sure whether mypy will like it.

Also it seems like the super classes should be able to accept `None`, since these arguments are passed on to them later on